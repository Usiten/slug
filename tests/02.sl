a = 8 / 4 / 2;
= a;