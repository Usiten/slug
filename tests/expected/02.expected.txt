-- Stack Top --
>1
---------------
-- Var Data --
a: 1
---------------
